designs/administrator/future-openshift/favoriting/index.md

```diff
@@ -3,29 +3,62 @@ parent: Administrator
 version: 4.x
 ---
 
-# Favoriting
+# Favoriting and hiding system projects
 
-## Projects list
-![1](img/projects1.png)
-* Users may set their favorite projects by clicking the star toggle at the beginning of a row.
-* The project will not change positions in the list when the star is toggled on or off.
+## Introduction
+Today, our project selector includes favoriting and setting default projects. PatternFly now has a favoriting component for both select menus and tables that we want to replace OpenShift's favoriting with. In addition, there are often many system projects in the project selector/project list view, which are unnecessary for most users. The following documentation explains how we will combine the effort of hiding system projects and favoriting. The design will be broken down into sections:
+1. [Favoriting and hiding system projects within the project selector](#Favoriting-and-hiding-system-projects-in-the-project-selector)
+2. [Favoriting and hiding system projects within the projects and namepsaces list views](#Favoriting-and-hiding-system-projects-in-projects-and-namespaces)
 
-![2](img/projects2.png)
-* Users may view only their favorite projects by checking the "Show only favorites" checkbox next to the textbox filter.
+## Favoriting and hiding system projects in the project selector
+The project selector will be replaced by PatternFly select with favorites so that it can make use of the newest PF favoriting functionality. We will remove the ability to default a project. A switch will be added to be able to hide and show system projects. The switch will be off by default, hiding system projects. A selected project will be reflected within the toggle and obtain a gray background within the menu (same as today). Lastly, we will move the Create project action to a footer within the menu so that the action is more obvious.
```

Comment:
Just to mention, we might want to update the Creating convention section '[Create buttons in selection fields](http://openshift.github.io/openshift-origin-design/conventions/documentation/create.html#create-buttons-in-selection-fields)' just because it was using this project selector example.

We should probably determine if this button at the bottom interaction is replacing the divider separated item everywhere (I don't think this is done more than 2 or 3 places right now) or if this is in addition and its up to the interaction which one to use.

Another example of where this create button appears in in the Namespace selector when installing an operator.

Reply:
It sounds like there was some previous user research done where users found it difficult to figure out how to create a project since the action looks like a menu item, so at least for the project selector, it should be updated. It may be worth coming up with designs for the namespace selector when installing an operator and showing to users to see if it helps in that context as well @itsptk